### PR TITLE
138 add ci to the smart contract example

### LINKED ISSUE #138: test smartContracts

### PR DESCRIPTION
**Pull request by Elli610**
closes #138

### CODE REVIEW

File: .github/workflows/deploy-test-smartContract.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18"
+          cache: "npm"
+          cache-dependency-path: ./examples/smartContracts/contracts/package-lock.json
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: build
```
Comment (gregLibert) [inline .github/workflows/deploy-test-smartContract.yml:29]:
```suggestion
      - name: Build
```

**Review by gregLibert — APPROVED** [review]

**Review by gregLibert — CHANGES_REQUESTED** [review]
Sorry bad action. Can you take into account my comment from the previous review ?

**Review by gregLibert — APPROVED** [review]

**Comment by gregLibert** [comment]
@Elli610 the CI has failed

**Comment by Ben-Rey** [comment]
@Elli610 The Ci need to be fixed

**Pull request merged**